src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -1209,7 +1209,7 @@ private void updateOverallHappiness()
             final BuildingHome home = citizen.getHomeBuilding();
             if(home != null)
             {
-                housing = home.getBuildingLevel();
+                housing += home.getBuildingLevel();
```

Comment:
If you have 3 Citizens but only 1 Citizens' hut level 2 (with Citizen A and B in it) 
You would have:
Citizen A +2
Citizen B +2
Citizen C +0
housing would be 4 when average >1 (4/3)
when housing should be 2 with average <1 (2/3) as in not very happy